src/App.tsx

```diff
@@ -67,18 +53,14 @@ export const App: React.FC = () => {
                   </div>
                 )}
 
-                {author && loaded && !hasError && posts.length === 0 && (
+                {author && loading && !err && posts.length === 0 && (
                   <div className="notification is-warning" data-cy="NoPostsYet">
                     No posts yet
                   </div>
                 )}
 
-                {author && loaded && !hasError && posts.length > 0 && (
-                  <PostsList
-                    posts={posts}
-                    selectedPostId={selectedPost?.id}
-                    onPostSelected={setSelectedPost}
-                  />
+                {author && loading && !err && posts.length > 0 && (
```

Comment:
The condition `loading && !err && posts.length > 0` might not be correct for displaying the `PostsList`. Typically, you would want to display the list after loading is complete, i.e., `!loading && !err && posts.length > 0`.